### DISCUSSION TITLE
The code coverage related packages in the unit test project...

### DISCUSSION THREAD

**Discussion by Numpsy** [icsharpcode/SharpZipLib#595] [General] [answered]
Something that I just remembered when looking at the packages referenced by the unit test project:

There are references to various code coverage related nuget packages in the unit test project (coveralls, opencover, report generator).

The package versions are rather out of date, and I don't think anything in the official builds use them, so I was just wondering if anything should be done with them - e.g., update to newer versions, remove them, make use of them and publish some results in the CI builds? (I don't know what the situation is with publishing coverage results into Github Actions builds though)

**Comment by piksel** [comment, marked as answer]
Yeah, good point. I removed the ones not in use, and added reporting + publishing to CodeCov in #602
